operator/pkg/values/translation/translation.go

```diff
@@ -123,40 +124,123 @@ func GetGlobalMonitoring(m *platform.GlobalMonitoring) *ValuesBuilder {
 	return &globalMonitoring
 }
 
-// SetScannerAnalyzerValues sets values in "sv" based on "analyzer".
-func SetScannerAnalyzerValues(sv *ValuesBuilder, analyzer *platform.ScannerAnalyzerComponent) {
-	if analyzer.GetScaling() != nil {
-		scaling := analyzer.GetScaling()
-		sv.SetInt32("replicas", scaling.Replicas)
-
-		autoscaling := NewValuesBuilder()
-		if scaling.AutoScaling != nil {
-			switch *scaling.AutoScaling {
-			case platform.ScannerAutoScalingDisabled:
-				autoscaling.SetBoolValue("disable", true)
-			case platform.ScannerAutoScalingEnabled:
-				autoscaling.SetBoolValue("disable", false)
-			default:
-				autoscaling.SetError(fmt.Errorf("invalid spec.scanner.replicas.autoScaling %q", *scaling.AutoScaling))
-			}
+// SetScannerComponentDisabledValue sets the disabled values for scanner configurations
+func SetScannerComponentDisabledValue(sv *ValuesBuilder, scannerComponent *platform.ScannerComponentPolicy) {
```

Comment:
Shouldn't this function be renamed to something more generic, as it should also be able to *enable* a scanner component? Depending on the provided component policy, I mean?